### DISCUSSION TITLE
Change formats for markups?

### DISCUSSION THREAD

**Discussion by rkrug** [mloubout/critic-markup#2] [Ideas]
First of a all a big thank you for this add-on - it is great!

Due to personal taste, I would like to have the removed text as strikethrough in addition to the red highlight - is this easily possible? And leading from that, I think it could be useful to be able to customise the used formats, e.g. in the yaml frontmatter of the document?

Cheers,

Rainer

**Comment by mloubout** [comment]
Hi

Glad you like it. 

At the moment the strikethrough/highlight are handled through a static css which makes it quite complicated (for me at least) to make it an option but I will put it on the list of future improvement as it seems to be a wanted feature. I cannot guaranty I will look at it soon as at the moment my priority is to work on rendering to pdf/latex/docx with the option to render the original/editied/diff version for publishing. Hopefully I can get to the fonts soon.

Cheers
